dev-docs/RFCs/v4.0.0/names_for_concepts_types_functions.md

```diff
@@ -66,6 +66,10 @@ The following technical terms should be used in the documentation, the H3 codeba
     - may include more than 6 points in the case that a cell is not a geometric hexagon, such as when a hexagon crosses an icosahedron boundary
     - may also be used to describe the boundary between two geometric cells, as in the case of an edge
     - represented in the H3 codebase with the `CellBoundary` struct (previously `GeoBoundary` before v4.0)
+- `H3_NULL`;
+    - equivalent to `0` and guaranteed to never be a valid `H3Index` (even after any future H3 **modes** are added)
```

Comment:
I’d suggest adding this in docs- the RFCs are more internal technical documentation and aren’t on the website

Reply:
It wasn't clear to me the best place to add this in the docs. Any ideas?

Reply:
Probably around the description of the H3 Index format in `docs/core-library/h3indexing.md`?

Reply:
I can add that in, after we figure out what to do with the `Mode 0` issue below.